client-src/elements/chromedash-feature-page.js

```diff
@@ -137,8 +141,13 @@ export class ChromedashFeaturePage extends LitElement {
         document.title = `${this.feature.name} - ${this.appTitle}`;
       }
       this.loading = false;
-    }).catch(() => {
-      showToastMessage('Some errors occurred. Please refresh the page or try again later.');
+    }).catch((error) => {
+      if (error instanceof FeatureNotFoundError) {
+        this.loading = false;
+        showToastMessage('Feature not found.');
```

Comment:
The toast displays for 7 seconds by default, but in this case I think we should have the message stay there until the user navigates to another page.

We could pass -1 for optDuration to make the toast stay, but then we need to add logic to make it disappear once the user navigates away.

I'm thinking that it would be better to show a message in the body of the SPA page.  Basically, on line 467, return an error message to display rather than using a toast for this situation at all.  In the issue, I mentioned the toast because it was showing the wrong message, but we're not bound to make the right message show in a toast.

Reply:
I changed it. It doesn't have any styling. But I think that could be for the new issue for the [common  error page](https://github.com/GoogleChrome/chromium-dashboard/issues/2257#issuecomment-1252700901)
![image](https://user-images.githubusercontent.com/7788930/205132890-2e24f1af-d8a5-49c6-87d8-e7112ca02312.png)